core/domain/question_domain_test.py

```diff
@@ -70,6 +165,48 @@ def test_strict_validation(self):
         self._assert_validation_error(
             'Expected at least one answer group to have a correct answer')
 
+    def test_strict_validation_for_answer_groups(self):
+        """Test to verify validate method of Question domain object with
+        strict as True for interaction with answer group.
+        """
+        state = self.question.question_state_data
+        state.interaction.default_outcome.labelled_as_correct = False
+        state.interaction.answer_groups = [
+            state_domain.AnswerGroup.from_dict({
+                'outcome': {
+                    'dest': 'abc',
+                    'feedback': {
+                        'content_id': 'feedback_1',
+                        'html': 'Feedback'
+                    },
+                    'labelled_as_correct': True,
+                    'param_changes': [],
+                    'refresher_exploration_id': None,
+                    'missing_prerequisite_skill_id': None
+                },
+                'rule_specs': [{
+                    'inputs': {
+                        'x': 'Test'
+                    },
+                    'rule_type': 'Contains'
+                }],
+                'training_data': [],
+                'tagged_misconception_id': None
+            })
+        ]
+
+        self._assert_validation_error(
+            'Expected all answer groups to have destination as None.')
+
+    def test_strict_validation_passes(self):
+        """Test to verify validate method of Question domain object with
+       strict as True with correct input.
```

Comment:
There isn't an explicit strict flag, right? Maybe change docstring to:
`Test to verify validate method of a finalized Question domain object with correct input`?